.eslintrc

```diff
@@ -1,76 +1,80 @@
 {
-  "parser": "babel-eslint", // now for the support of allowImportExportEverywhere
+  "parser": "babel-eslint",
+  // "extends": [
+  //   "plugin:jsx-a11y/recommended"
```

Comment:
This pulls in default accessibility recommendations for React components, which I think is a requirement for Reaction, right? This, like the rules, can be uncommented in a later ticket. There are ~100 errors from it, and they'll take manual work to solve.